app/stylesheet/miq-structured-list.scss

```diff
@@ -1,12 +1,140 @@
 .miq-structured-list {
-  border: 1px solid lightgray;
   margin-bottom: 2rem !important;
 
+  .list_cell {
+    padding: 5px 0 5px 0;
+  }
+
+  .clickable {
+    cursor: pointer;
+
+    &:hover {
+      background: #f4f4f4;
+    }
+  }
+
+  .list_header {
+    .sortable_header {
+      display: flex;
+    }
+
+    &.bx--structured-list-th {
+      padding: 0.8em 0.5rem 0.5rem 0.3rem !important;
```

Comment:
@jeffibm I am seeing many important tags in css. Are they needed 100%. Writing important tags is not a good practice unless it is absolutely required.